javascript-injectors/README.md

```diff
@@ -39,52 +37,63 @@ response = requests.patch(
 )
 ```
 
-## Several JS injection examples
+### Important considerations
 
+* Your script should be idempotent as here is a possibility it will be run several times.
+* Your should execute with interval or awaiting when certain element appear on page. There is no guarantee that page is properly load when script is run.
+* If you are setting cookies, they will persists while playlist is being run. As such, you can optimize your script by checking if cookie is already set.
+* In case your page includes redirects, the injection script will be executed on each page.
 
-<img src="../images/screenly-logo.png" alt="Screenly logo" width="200"/>
 
-## Sign in to Screenly webconsole via cookies
+## Examples
 
-#### Log in to the console
 
-#### Required cookies 
-- `beaker.session.id`
+### Scroll down
 
-#### [JavaScript Injector](https://github.com/Screenly/playground/tree/master/javascript-injectors/screenly-signin-via-cookies.js)
+Perhaps the most simple example would be to scroll down a page. We can accomplish this by just using the `window.scrollBy()` function. For instance, if we want to scroll down 180 pixels, we can use:
 
----
+```javascript
+window.scrollBy(0,180);
+```
 
-<img src="../images/tableau-logo.png" alt="Tableau logo" width="200"/>
+For more details see the `window.ScrollBy()` [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollBy).
 
-## Sign in to Tableau via cookies
 
-#### Log in to the console
+### Sign in to Screenly via cookies
 
-#### Required cookies
-- `SSESS...`
+<img src="../images/screenly-logo.png" alt="Screenly logo" width="200"/>
+
+* Sign into the Screenly [user interface](https://login.screenlyapp.com).
+* Retrieve the cookie `beaker.session.id` from your browser.
+* Download [screenly-signin-via-cookies.js](https://github.com/Screenly/playground/tree/master/javascript-injectors/screenly-signin-via-cookies.js) and modify it with your `beaker.session.id`.
+* PATCH the asset.
 
-#### [JavaScript Injector](https://github.com/Screenly/playground/tree/master/javascript-injectors/tableau-via-cookies.js)
 
----
+### Sign in to Tableau via cookies
 
-<img src="../images/powerbi-logo.png" alt="PowerBi logo" width="200"/>
+<img src="../images/tableau-logo.png" alt="Tableau logo" width="200"/>
 
-## Sign in to PowerBi via cookies
+* Sign into your Tableau account.
+* Extract the cookies `SSESS...` from your browser.
```

Comment:
@rusko124 Can you clarify on this for me? Below we have all cookies specified. It's unclear to me what `SSESS..` means.

Reply:
The name of the key starts with SSESS
Example: `SSESSf61f10f5bab267d95b12f296c07a7e69`